src/SFA.DAS.EmployerIncentives.Abstractions/Commands/IScheduledCommandPublisher.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.EmployerIncentives.Abstractions.Commands
+{
+    public interface IScheduledCommandPublisher
```

Comment:
Do we need a separate interface?  Can't the method go on the ICommandPublisher and the delay is implied by the method arguments,

Reply:
I started off with it on ICommandPublisher but they have different dependencies so it felt a bit odd once I'd done it.